code/modules/admin/verbs/debug.dm

```diff
@@ -748,6 +748,38 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 		if("Respawnable Mobs")
 			to_chat(usr, jointext(respawnable_list,","))
 
+/client/proc/cmd_display_del_log()

```

Comment:
This one is insanely detailed, which is great, but sometimes you don't care about this level of input.

I'm tempted to code a secondary version that just lists things that failed, don't respect force, sleep in the destroy, and have no hint.

The old version was really easy to see at a glance without scrolling through every single thing deleted in the entire game, ever.